### PR TITLE
Feature add contract for journey accounting and shuffle ResponsibilitySet

### PR DESCRIPTION
**Pull request by skinkie**
This pull request, based on @nick-knowles commit,mixes contract stuff with a modification of ResponsibilitySet towards different files.

In addition I have fixed the two refered examples so they don't include "xsi" attributes, and correctly refer to nameOfRefClass.

### CODE REVIEW

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+			<xsd:enumeration value="secures"/>
+			<xsd:enumeration value="redistributes"/>
+			<xsd:enumeration value="supports"/>
+			<xsd:enumeration value="owns"/>
+			<xsd:enumeration value="other"/>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<xsd:simpleType name="DataRoleTypeListOfEnumerations">
+		<xsd:annotation>
+			<xsd:documentation>List of Allowed values for Data e Roles.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:99]:
```suggestion
			<xsd:documentation>List of Allowed values for DataRoles.</xsd:documentation>
```

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+		</xsd:simpleContent>
+	</xsd:complexType>
+	<xsd:element name="AllOrganisationsRef" type="AllOrganisationsRefStructure" substitutionGroup="VersionOfObjectRef">
+		<xsd:annotation>
+			<xsd:documentation>Reference to all ORGANISATIONs.</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="AllOrganisationsRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a reference tto all oranisationsORGANISATION.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:168]:
```suggestion
			<xsd:documentation>Type for a reference to all ORGANISATIONs.</xsd:documentation>
```

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+			<xsd:enumeration value="formal"/>
+			<xsd:enumeration value="informal"/>
+			<xsd:enumeration value="written"/>
+			<xsd:enumeration value="oral"/>
+			<xsd:enumeration value="plainUnderstood"/>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<xsd:simpleType name="LegalStatusEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for Legal Status of a Contract</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:288]:
```suggestion
			<xsd:documentation>Allowed values for Legal Status of a Contract.</xsd:documentation>
```

File: xsd/netex_framework/netex_responsibility/netex_responsibility_support.xsd [hunk 4]
```diff
@@ -85,169 +86,7 @@ Rail transport, Roads and Road transport
 			</xsd:restriction>
 		</xsd:simpleContent>
 	</xsd:complexType>
-	<!-- =====AdministrativeRole Identifiers=============================================================== -->
-	<xsd:simpleType name="StakeholderRoleTypeEnumeration">
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibility_support.xsd:89]: For reference. This went into ./netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:

File: xsd/netex_framework/netex_responsibility/netex_responsibility_support.xsd [hunk 5]
```diff
@@ -268,84 +107,4 @@ Rail transport, Roads and Road transport
 		</xsd:simpleContent>
 	</xsd:complexType>
 	<!-- ======================================================================= -->
-	<xsd:simpleType name="ResponsibilityRoleIdType">
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibility_support.xsd:271]: For reference: this went into ./netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:

File: xsd/netex_framework/netex_reusableComponents/netex_transportOrganisation_support.xsd [hunk 6]
```diff
@@ -14,11 +14,11 @@
...
 					<Modified>2014-03-31</Modified>
 				</Date>
-				<Date><Modified>2019-03-08</Modified>UK-44 Add additional  groupsOfOperators  to Network.   
+				<Date><Modified>2019-03-08</Modified>UK-44 Add additional  groupsOfOperators  to Network.
 				</Date>
-				<Date><Modified>2020-10-15</Modified>NewModes: Add TransportOperator PublicTransportOperator, Add alternativeTransportOperatorrk.   
+				<Date><Modified>2020-10-15</Modified>NewModes: Add TransportOperator PublicTransportOperator, Add alternativeTransportOperator.
 				</Date>
-				<Date><Modified>2020-11-24</Modified>NewModes: Drop  PublicTransportOperator,   alternativeTransportOperatorrk.   
+				<Date><Modified>2020-11-24</Modified>NewModes: Drop PublicTransportOperator,   alternativeTransportOperator.
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_transportOrganisation_support.xsd:21]:
```suggestion
				<Date><Modified>2020-11-24</Modified>NewModes: Drop PublicTransportOperator, alternativeTransportOperator.
```

**Review by ue71603 — APPROVED** [review]

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd [hunk 7]
```diff
@@ -468,17 +479,5 @@ Rail transport, Roads and Road transport
 			</xsd:extension>
 		</xsd:complexContent>
 	</xsd:complexType>
-	<xsd:complexType name="contractsInFrame_RelStructure">
```
[the remark this answers is not in the document]
Comment (ue71603) [reply xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd:471]: @Aurige @nick-knowles   If ok for you, I swap to lower case.

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+			<xsd:enumeration value="Operation"/>
+			<xsd:enumeration value="Control"/>
+			<xsd:enumeration value="Reservation"/>
+			<xsd:enumeration value="EntityLegalOwnership"/>
+			<xsd:enumeration value="FareManagement"/>
+			<xsd:enumeration value="Financing"/>
+			<xsd:enumeration value="SecurityManagement"/>
+			<xsd:enumeration value="CustomerService"/>
+			<xsd:enumeration value="DataRegistrar"/>
+			<xsd:enumeration value="Other"/>
```
[the remark this answers is not in the document]
Comment (ue71603) [reply xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:69]: @Aurige @nick-knowles  Shall I add the roles?

[the remark this answers is not in the document]
Comment (ue71603) [reply xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:69]: @Aurige should we add the roles André mentioned?

[the remark this answers is not in the document]
Comment (ue71603) [reply xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:69]: added

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd [hunk 9]
```diff
@@ -0,0 +1,296 @@
...
+		<xsd:documentation>RESPONSIBILITY i.e. Data administration identifier types for NeTEx.</xsd:documentation>
+	</xsd:annotation>
+	<xsd:include schemaLocation="netex_relationship.xsd"/>
+	<!-- ===== ADMINISTRATIVE ROLE =========================================================== -->
+	<xsd:simpleType name="StakeholderRoleTypeEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for Stakeholder Roles.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:string">
+			<xsd:enumeration value="Planning"/>
```
[the remark this answers is not in the document]
Comment (ue71603) [reply xsd/netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd:59]: done to lower

**Review by trurlurl — APPROVED** [review]

**Pull request merged**